content/en/security/application_security/threats/setup/_index.md

```diff
@@ -1,29 +1,4 @@
 ---
 title: Threat Management Setup
-disable_toc: false
+external_redirect: /security/application_security/threats/threat_detection/
```

Comment:
@robertpi why are you adding this redirect?

The current pages are:

- [Setup](https://docs.datadoghq.com/security/application_security/threats/setup/)
  - [Library Compatibility](https://docs.datadoghq.com/security/application_security/threats/setup/compatibility/)
  - [Using Single Step Instrumentation](https://docs.datadoghq.com/security/application_security/threats/setup/single_step/)
  - [Using Datadog Tracing Libraries](https://docs.datadoghq.com/security/application_security/threats/setup/threat_detection/)

We can just remove **Using Single Step Instrumentation** and references to it without deleting the content of its parent **Setup**.

BTW, redirects are done using an `aliases` label not `external_redirect`.

Reply:
@robertpi agreed in Slack.